frontend/src/components/feedback/FeedbackDialog.tsx

```diff
@@ -70,14 +70,13 @@ export default function FeedbackDialog({ onClose, open, title, inputLabel, maxLe
             value={element}
             inputProps={{ maxLength: maxLength }}
             rows={4}
-            rowsMax={15}
+            maxRows={15}
             required
           />
           <Checkbox
             id={"feedbackcheckbox"}
             className={classes.checkbox}
             checked={checked}
-            color="primary"
```

Comment:
Again, why did you remove the color from the checkbox?

Reply:
From Pr description step 8: Checkbox: In mui 5 the default color for Checkbox is now primary. Since all our Checkboxes had color="primary" anyways I removed all the explicit color=primary statements as they are now redundant.